docs/mkdocs/api/authentication.md

```
@@ -0,0 +1,162 @@
+
+```json
+{
+  "status": "success",
+  "data": "true"
+  "message": "Login successful"
+}
+```
+
+The response includes a `Set-Cookie` header with an HTTP-only cookie named `access_token` containing the JWT. Save this cookie (e.g., in `cookies.txt`) for subsequent requests.
+
+!!! Tip 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)